### PR TITLE
COMMERCE-2488 Added capability to pass subscription ifno

### PR DESCRIPTION
**Pull request by alessiorendina**

### CODE REVIEW

**Comment by alessiorendina** [comment]
ci:test

**Review by ethanbustad — APPROVED** [review]

File: commerce-initializer-util/src/main/java/com/liferay/commerce/initializer/util/CPDefinitionsImporter.java [hunk 1]
```diff
@@ -189,7 +191,8 @@ private CPDefinition _addCPDefinition(
 			long catalogGroupId, String name, String shortDescription,
 			String description, String externalReferenceCode, String sku,
 			String taxCategory, long width, long height, long depth,
-			long weight, long[] assetCategoryIds, String[] assetTagNames,
+			long weight, JSONObject subscriptionInfoJSONObject,
```
Comment (marco-leo) [inline commerce-initializer-util/src/main/java/com/liferay/commerce/initializer/util/CPDefinitionsImporter.java]: @Commando18 Please avoid to pass JSONObject, better pass all fields needed

Comment (alessiorendina) [reply commerce-initializer-util/src/main/java/com/liferay/commerce/initializer/util/CPDefinitionsImporter.java]: Fixed, thanks!

**Comment by alessiorendina** [comment]
ci:test

**Pull request merged**